### ISSUE TITLE
Constant background Bluetooth time exceeded messages on Series 5 watch

### ISSUE THREAD

**Issue by Steve-gnome** [Kemmey/Kemmey-TeslaWatch-Public#567]
Following latest upgrade Latest version now generates constant background Bluetooth warning messages.  I understand the background stuff doesn’t work on series 5 but previously app was great now it doesn’t work because of the constant nag

**Comment by Yonno13** [comment]
I am experiencing the same and it is VERY ANNOYING. I have a Series 4 and it was working very well. How can I get these messages to stop!

**Comment by Kemmey** [comment]
1.3.2 was submitted to apple yesterday, fixing this. It is STILL awaiting app review... it will release as soon as apple gets their act together and approves the build...

**Comment by calvarez** [comment]
Funny, I got a notification of this post as I was driving and getting a ton of these. Mine only happened while driving, so far.

**Comment by tyrotex** [comment]
Series 4 watch; v1.3.1 WAfT; M3 v2023.6.11
I've turned off Bluetooth control and I'm still being bombarded with messages since I used the app once today.
Error in logs is "Error updating BLE value: Exceeded number of Background Notification" every 1-3 minutes.
Logs sent.

**Comment by Kemmey** [comment]
Yeah - they're taking an exceedingly long amount of time to approve this fix. Been 38 hours now...

**Comment by Kemmey** [comment]
FINALLY - app has entered review... should be no more than an hour now... 🤞🤞🤞

**Comment by Kemmey** [comment]
1.3.2 has been released to the store! Start your update checks! So sorry for the disruption!

**Comment by tyrotex** [comment]
Thanks Kim.
AU app store is still serving 1.3.1.
Hopefully, it will update soon.

**Comment by molster** [comment]
Im on the latest version of the app, (and watch use to be able to open the door all on its own) however now it appears to be infantility stuck on "connection" when looking at it in the settings

I can confirm im on build 1.3.2  build 1274 

any trick I need to do after doing the update?

**Comment by molster** [comment]
any updates on this? Just been stuck on connection ever since the update

**Comment by Kemmey** [comment]
@molster dont see how this is related to the very much solved issue of notifications?!

Try removing the watch as key in the vehicle locks menu - and then pair again?
